modules/ztp-install-factory-hubspoke-architecture.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="hub-edge-cluster-architecture_{context}"]
+= The hub and edge cluster architecture
```

Comment:
Looked odd for a minute starting with "The". Could it just be "Hub and edge cluster architecture"?